src/ext/browserUtils.js

```diff
@@ -49,25 +48,22 @@ function attemptRedeem(url, respTabId, target) {
                 if (tabIds.includes(respTabId)) {
                     chrome.cookies.get({
                         "url": url.href,
-                        "name": CHL_CLEARANCE_COOKIE,
-                        "storeId": store.id,
-                    }, function(cookie) {
                         // Require an existing, non-expired cookie.
-                        if (cookie) {
+                        if(cookie) {
```

Comment:
Can we add this space back?

Reply:
ESLint complains about `(space-before-function-paren)`, that seems wrong, I'll look into it